app/routes/auth/signin-oidc.js

```diff
@@ -76,8 +85,33 @@ module.exports = [{
           case err instanceof InvalidStateError:
```

Comment:
instead of the let above we maybe could create a `const` here

`
const event = {
          id: request.yar.id,
          sbi: organisationSummary.organisation.sbi,
          crn: organisationSummary.organisation.crn,
          exception: err.name
        }
`